contracts/emissions/BasicRewardsForwarder.sol

```diff
@@ -29,13 +31,11 @@ contract BasicRewardsForwarder is
     event RecipientChanged(address newRecipient);
 
     /**
-     * @dev Constructor
      * @param _nexus        mStable system Nexus address
      * @param _rewardsToken Token that is being distributed as a reward. eg MTA
      */
     constructor(address _nexus, address _rewardsToken)
-        InitializableRewardsDistributionRecipient(_nexus)
-    {
+        InitializableRewardsDistributionRecipient(_nexus) {
```

Comment:
Husky commitlint still not running or we have mismatching configs 🤔  weird

Reply:
I forgot to install husky on my new global node version.